notebooks/p-center.ipynb

```diff
@@ -0,0 +1,914 @@
+       "       [ 8.05520721,  3.2777533 ,  5.1127178 ,  6.99019592, 12.31431023],\n",
+       "       [ 8.033197  ,  3.2997635 ,  5.09070759,  6.96818571, 12.29230002],\n",
+       "       [ 4.88391014,  5.94387041,  3.55339931,  1.6759212 ,  4.62480712],\n",
+       "       [ 3.38092176,  9.44685879,  6.32341117,  5.17890958,  7.85479527],\n",
+       "       [ 5.83945489,  5.17241539,  2.78194429,  0.90446618,  4.41964814],\n",
+       "       [10.25764123,  4.57013932,  5.82033178,  3.69780989,  5.02192421],\n",
+       "       [ 3.16471551,  8.168245  ,  1.7777739 ,  3.90029578,  7.59956747],\n",
+       "       [ 8.83620663,  8.49675387,  5.89371722,  7.77119534, 13.09530965],\n",
+       "       [ 7.60754658,  6.94050708,  4.55003599,  2.67255787,  2.65155644],\n",
+       "       [ 4.14555919,  9.4785197 ,  7.0880486 ,  5.21057048,  5.09015784],\n",
+       "       [ 7.24126831,  4.57422881,  2.80395885,  0.68143697,  5.01783472],\n",
+       "       [ 5.70322513,  8.53100569,  2.76073572,  4.63821384,  9.96232815],\n",
+       "       [ 9.27617639,  9.55160416,  7.16113307,  5.28365495,  0.04045936],\n",
+       "       [ 2.5651854 , 11.39296595,  5.00249486,  7.12501674,  9.80090243],\n",
+       "       [14.22296519,  3.5559257 ,  9.78565573,  7.66313385,  6.03613783],\n",
+       "       [ 8.33806089,  2.48971967,  3.90075143,  5.77822955, 11.10234386],\n",
+       "       [14.34079531,  3.51301476,  9.90348585,  7.78096397,  7.91830771],\n",
+       "       [ 7.55811406,  6.89107456,  4.50060346,  2.62312535,  2.70098897],\n",
+       "       [ 9.54667188,  8.87963238,  6.48916129,  4.61168317,  0.71243114],\n",
+       "       [ 6.99771477,  3.83006578,  2.56040532,  2.43788343,  7.76199775],\n",
+       "       [10.85478728,  4.18774778,  6.41747782,  4.29495594,  5.40431574],\n",
+       "       [ 6.89563349,  8.43732701,  3.95314408,  5.8306222 , 11.15473651],\n",
+       "       [12.29945454,  3.63241504,  7.86214508,  5.7396232 ,  5.95964848],\n",
+       "       [ 6.57929244,  6.75366806,  3.63680304,  5.51428115, 10.83839547],\n",
+       "       [ 8.35675866,  8.47102189,  6.0805508 ,  4.20307268,  1.90234436],\n",
+       "       [11.26183   ,  1.40520949,  6.82452055,  5.67315871, 10.99727302],\n",
+       "       [ 6.92663397,  8.25959447,  5.86912337,  3.99164526,  2.30908306],\n",
+       "       [ 6.97410775,  3.8536728 ,  2.53679829,  3.96088096,  9.28499527],\n",
+       "       [10.00715257,  8.82062799,  6.94964198,  4.92783614,  0.77143554],\n",
+       "       [ 8.83013405,  7.9976465 ,  5.77262346,  3.89514534,  1.59441703],\n",
+       "       [ 6.69445759,  4.63850292,  2.86823295,  4.74571107, 10.06982539],\n",
+       "       [ 2.60649588, 11.43427644,  5.04380534,  7.16632722,  9.84221291],\n",
+       "       [ 9.01806225,  4.31489826,  6.07557284,  7.95305096, 13.27716527],\n",
+       "       [ 7.49191577,  3.84104474,  4.07077477,  5.94825289, 11.2723672 ],\n"
```

Comment:
I can see that you changed P-Median to P-Center in cell 19, but the plots are still labeled P-Median in the output of cell 20 and 21. Maybe let's try to rerun again to fix this?

Also, go ahead and fully rerun `facloc-real-world.ipynb` (the *p*-median plots seem to have not been run). And 👍 👍 for the solver comparison at the end of `facloc-real-world.ipynb`. Very interesting stuff. I'm wondering if maybe we should include solve time in the print out for comparison. Maybe I'll include that later.